net-irc/anope/anope-2.0.7.ebuild

```diff
@@ -0,0 +1,142 @@
+DEPEND="
+	acct-group/anope
+	acct-user/anope
+	mysql? ( dev-db/mysql-connector-c:0= )
+	ssl? ( dev-libs/openssl:0= )
+	gnutls? ( net-libs/gnutls:0= dev-libs/libgcrypt:0= )
+	ldap? ( net-nds/openldap )
+	pcre? ( dev-libs/libpcre )
+	sqlite? ( dev-db/sqlite:3 )
+	tre? ( dev-libs/tre )
+	virtual/libintl
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/pid-patch.patch" )
+
+S="${WORKDIR}/${P}-source"
+
+src_prepare() {
+	anope_enable_mod() {
+		local modulefile=${1}
+		ln -s "extra/${modulefile}" "modules/" || \
+			die "Failed to use ${modulefile}"
+	}
+
+	# These all require MySQL specifically
+	if use mysql; then
+	   anope_enable_mod "m_mysql.cpp"
+	   anope_enable_mod "stats/irc2sql/irc2sql.cpp"
+	   anope_enable_mod "stats/irc2sql/irc2sql.h"
+	   anope_enable_mod "stats/irc2sql/tables.cpp"
+	   anope_enable_mod "stats/irc2sql/utils.cpp"
+	   anope_enable_mod "stats/m_chanstats.cpp"
+	   anope_enable_mod "stats/cs_fantasy_top.cpp"
+	   anope_enable_mod "stats/cs_fantasy_stats.cpp"
+	   anope_enable_mod "m_sql_log.cpp"
+	   anope_enable_mod "m_sql_oper.cpp"
+	fi
+
+	use sqlite && anope_enable_mod "m_sqlite.cpp"
+
+	# Any SQL implementation
+	if use sql; then
+	   anope_enable_mod "m_sql_authentication.cpp"
+	   anope_enable_mod "m_sql.cpp"
+	   anope_enable_mod "m_sql_live.cpp"
+	fi
+
+	if use ldap; then
+		anope_enable_mod "m_ldap.cpp"
+		anope_enable_mod "m_ldap_authentication.cpp"
+		anope_enable_mod "m_ldap_oper.cpp"
+	fi
+
+	use gnutls && anope_enable_mod "m_ssl_gnutls.cpp"
+	use pcre && anope_enable_mod "m_regex_pcre.cpp"
+	use ssl && anope_enable_mod "m_ssl_openssl.cpp"
+	use tre && anope_enable_mod "m_regex_tre.cpp"
+
+	# Unconditional modules
+	anope_enable_mod "m_regex_posix.cpp"
+
+	# Avoid a silly sandbox error - tries to delete /usr/lib/modules
+	sed -i '/install.*REMOVE_RECURSE.*/d' CMakeLists.txt || die
+
+	# Copy anope.conf for fixup to comply w/ prefix
+	cp "${FILESDIR}/anope-conf.d-v2" "${T}" || die
+
+	# Look in the right place for modules
+	sed -i "s~%LIBDIR%~${EPREFIX}/usr/$(get_libdir)/anope/~" \
+		"${T}/anope-conf.d-v2" || die
+
+	# TODO: May need to move this above
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBIN_DIR="libexec/anope"
+		-DDB_DIR="../var/lib/anope"
+		-DDOC_DIR="share/doc/${PF}"
+		-DLIB_DIR="$(get_libdir)/anope"
+		-DLOCALE_DIR="share/locale"
+		-DCONF_DIR="/etc/anope"
+		-DLOGS_DIR="../var/log/anope/"
+	)
+
+	cmake_src_configure
+}
+
+src_install() {
+	cmake_src_install
+
+	newinitd "${FILESDIR}/anope-init.d" "${PN}"
+	newconfd "${T}/anope-conf.d-v2" "${PN}"
+
+	dosym ../libexec/anope/services /usr/bin/services
+	dosym ../libexec/anope/anopesmtp /usr/bin/anopesmtp
+
+	keepdir /var/log/anope /var/lib/anope/backups
+	fowners anope:anope /var/{lib,log}/anope /var/lib/anope/backups
+
+	dodoc -r docs/* data/example.conf
+
+	insinto /etc/anope
+	newins data/example.conf services.conf
+
+	# TODO: Run Anope as root and have it drop down
+	fowners root:root /etc/anope
```

Comment:
This should be the default, what happens in this directory for it to require `anope` user and group access? You have `/var` for that reason usually.